app/src/main/kotlin/compiler/lowlevel/allocation/SpillsHandlingAllocation.kt

```diff
@@ -1,6 +1,12 @@
-package compiler.intermediate_form
-
-import compiler.common.intermediate_form.Allocation
+package compiler.lowlevel.allocation
+
+import compiler.intermediate.Register
+import compiler.intermediate.generators.memoryUnitSize
+import compiler.lowlevel.Addressing
+import compiler.lowlevel.Asmable
+import compiler.lowlevel.Instruction
+import compiler.lowlevel.Label
+import compiler.lowlevel.dataflow.Liveness
 
 object SpillsHandlingAllocation {
```

Comment:
Now, it looks a little weird that `SpillsHandlingAllocation` is not an instance of `Allocation`. Maybe we should add an additional interface for this and move all the coloring staff into a nested pachage `partialallocation`. (And also change `Allocation` to `PartialAllocation`) And maybe also change `SpillsHandlingAllocation` to `Allocation` or `FullAllocation`

Reply:
Renamed `Allocation` into `PartialAllocation` and `SpillsHandlingAllocation` into `Allocation`.
Unrelatedly, also renamed `LexerRegexParser` into `TokenRegexParser`.

There is *a lot* more inconsistencies in this project to fix, and this PR only deals with file names.